### ISSUE TITLE
mobile - locations - accordion interactivity implementation

### ISSUE THREAD

**Issue by IdaAkiwumi** [religious-ecologies/omeka-s-theme#56]
The states on this page seem to act as accordions when tapped, but they don't close when clicked again. Also, when the cities within them are tapped, nothing on the page alerts the user that more information for that particular city/state has been populated below. 

On the browser; however, displaying of related info is no issue since the information is populated to the right of the cities/states. To resolve the mobile issue, I think the states should be accordion styled to ensure the user can quickly flatten cities from a particular state and open another state link. 

I'm thinking for a page alert to show the user that information has been populated below, text like "Alamosa, Colorado info below" could be displayed to the right of the link that was pressed (at least on mobile). Another idea is displaying a down arrow link to the right of the selected city, and then when the user taps it, it shifts down the page to the start of the additional info section...I'm not sure if this can be done without it showing on the browser version.

<img width="365" alt="Screen Shot 2020-11-11 at 12 30 11 PM" src="https://user-images.githubusercontent.com/618307/98844037-c2728700-2419-11eb-8f3b-624c37f9a5c7.png">

**Comment by gswain2** [comment]
This is the same thing I was referring to in issue #55. Thanks Ida for explaining it better that I did!
